app/src/main/java/com/keylesspalace/tusky/components/compose/dialog/CaptionDialog.kt

```diff
@@ -50,9 +52,9 @@ fun <T> T.makeCaptionDialog(existingDescription: String?,
     dialogLayout.setPadding(padding, padding, padding, padding)
 
     dialogLayout.orientation = LinearLayout.VERTICAL
-    val imageView = TouchImageView(this).apply {
-        maxZoom = 6f
-    }
+    val imageView = BigImageView(this)
+    // imageView.ssiv.maxScale = 6f
```

Comment:
Woops. TODO for myself. ssiv is not initialized before we have an image, so it must be put in "onSuccess" method of ImageLoaderCallback.